src/Result.php

```diff
@@ -11,32 +11,37 @@
 class Result
 {
     /**
-     * @var string UA-Parser element used for match: 'device', 'os', 'ua'
+     * @var string
      */
-    public $matchType;
+    public $matchClass;
```

Comment:
`private` ?

Reply:
I thought about this, but it's accessed externally via _ClassifierTest_ on line 54 for the assertion message.

Not sure whether it's worth keeping `public` for this, switching it to `private` and adding a `getMatchClass()` method, or just removing the assertion message altogether.

Reply:
👍 for `getMatchClass()`